pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[build-system]
+requires = [ "cmake~=3.24.0", "ninja; platform_system!='Windows'", "setuptools>=42", "toml",]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "PennyLane_Lightning"
+description = "PennyLane-Lightning plugin"
+readme = "README.rst"
+requires-python = ">=3.9"
+classifiers = [ "Development Status :: 4 - Beta", "Environment :: Console", "Intended Audience :: Science/Research", "License :: OSI Approved :: Apache Software License", "Natural Language :: English", "Operating System :: MacOS :: MacOS X", "Operating System :: Microsoft :: Windows", "Operating System :: POSIX", "Operating System :: POSIX :: Linux", "Programming Language :: Python", "Programming Language :: Python :: 3", "Programming Language :: Python :: 3 :: Only", "Programming Language :: Python :: 3.9", "Programming Language :: Python :: 3.10", "Programming Language :: Python :: 3.11", "Programming Language :: Python :: 3.12", "Topic :: Scientific/Engineering :: Physics",]
+dependencies = [ "pennylane>=0.37",]
+dynamic = [ "version",]
+[[project.maintainers]]
```

Comment:
Double brackets implies it belongs to the previous section?

Reply:
Technically yes, Formally, double brackets mean that we are appending a table to an array of tables.
We have more than one way of doing that and when we dump the project file the toml package makes the decisions regarding how it will represent the data structure in a file. Same as the classifiers question before that.